src/object/explosion.cpp

```diff
@@ -90,9 +90,10 @@ Explosion::explode()
 
       /* The force decreases with the distance squared. In the distance of one
        * tile (32 pixels) you will have a speed increase of 150 pixels/s. */
-      float force = 150.0f * 32.0f * 32.0f / (distance * distance);
-      if (force > 200.0f)
-        force = 200.0;
+      float force = 150.0f * 64.0f * 64.0f / (distance * distance);
+	  // If we somehow get a force of over 500, keep it at 500 because unexpected behaviour could result otherwise.
+	  if (force > 500.0f)
```

Comment:
I can see inconsistent indentation here (maybe tabs need to be changed to spaces).